trulens_eval/tests/unit/test.py

```diff
@@ -42,29 +56,257 @@ def requiredonly_test(testmethodorclass):
 
 
 def module_installed(module: str) -> bool:
+    """Check if a module is installed."""
+
     try:
         __import__(module)
         return True
     except ImportError:
         return False
 
 
+T = TypeVar("T")
+
+
+def hashable_skip(obj: T, skips: Set[str]) -> T:
+    """Return a hashable copy of `obj` with all keys/attributes in `skips`
+    removed. 
+    
+    Sequences are returned as tuples and container types are returned as
+    frozendicts. Floats are retuned as 0.0 to avoid tolerance issues. Note that
```

Comment:
returned

Reply:
Also, shouldn't we be checking floats are the same? If you're worried about tolerance issues, why don't you just check with an allowed tolerance?

Reply:
See the docstring; this is not used for comparing values which is done with tolerance somewhere else. This is only used for sorting.

Reply:
Setting to 0 here effectively means that floats have no impact on sorting.